packages/web-api/src/methods.ts

```diff
@@ -2077,6 +2087,19 @@ export interface FilesRemoteShareArguments extends WebAPICallOptions, TokenOverr
   external_id?: string;
 }
 
+/*
+ * `functions.*`
+ */
+export interface FunctionsCompleteErrorArguments extends WebAPICallOptions, TokenOverridable {
+  function_execution_id: string;
+  error: string;
+}
+
+export interface FunctionsCompleteSuccessArguments extends WebAPICallOptions, TokenOverridable {
+  function_execution_id: string;
+  outputs?: Record<string, unknown>;
```

Comment:
I believe this paramter is required (our work-in-progress document says so too)

Reply:
You're probably right. I'll retest to be doubly certain and remove optionality upon confirmation.

Reply:
Although, to be fair, `outputs` may not be passed on so it's odd to require them!

Reply:
if i remember correctly, an empty `outputs` object might be required even when the function does not have any required output_parameters in its definition.

Reply:
Worth testing as you can also set `complete:false` on a function in the case it has interactivity sub-steps involved (such as the function creates a modal, which requires user input and interaction).

Reply:
Indeed, on the deno side, `outputs` is a required property (see [this code](https://github.com/slackapi/deno-slack-api/blob/main/src/typed-method-types/functions.ts#L3-L5)).